setup.py

```diff
@@ -202,6 +202,25 @@ def can_import(module_name):
     Extension("Bio.PDB.kdtrees", ["Bio/PDB/kdtrees.c"]),
 ]
 
+USE_MYPYC = False
+# To compile with mypyc, a mypyc checkout must be present on the PYTHONPATH
+if len(sys.argv) > 1 and sys.argv[1] == "--use-mypyc":
```

Comment:
This requires that `--use-mypyc` be the first option. It might be nice to detect it anywhere within sys.argv.

Of course, the ideal would be if setuptools provided a way to robustly add arguments. However I've experimented with custom cmdclass for this purpose, and it caused all kinds of trouble. I wouldn't recommend it for the main install command. So munging sys.argv seems ok, but maybe make it more generic.

Reply:
I'll take it out in favor of the environment variable approach.